package.json

```diff
@@ -27,7 +28,9 @@
     "tailwindcss": "^1.8.6",
     "@fortawesome/fontawesome-svg-core": "^1.2.31",
     "@fortawesome/free-solid-svg-icons": "^5.15.0",
-    "@fortawesome/react-fontawesome": "^0.1.11"
+    "@fortawesome/react-fontawesome": "^0.1.11",
+    "react": "^>=16.14.0",
+    "react-dom": ">=16.14.0"
```

Comment:
This is tangential to the main changes, but specified react and react-dom as peer deps. Using dbf-cloud's version as the minimum.